app/views/events/_form.html.erb

```diff
@@ -0,0 +1,40 @@
+<%= form_with model: @event do |form| %>
+  <div>
+    <%= form.label :title %><br>
+    <%= form.text_field :title %>
+    <% @event.errors.full_messages_for(:title).each do |message| %>
```

Comment:
Consider moving the error message display logic to a separate partial or helper method to keep the form code clean and readable.